server/handlers.go

```diff
@@ -69,9 +69,10 @@ func (cs *cribbageServer) createPlayer(username, name string) (model.Player, err
 	return mp, nil
 }
 
-func (cs *cribbageServer) setInteraction(pID model.PlayerID, im model.InteractionMeans) error {
-	ip := interaction.New(pID, im)
-	return cs.db.SaveInteraction(ip)
+func (cs *cribbageServer) setInteraction(pID model.PlayerID, im interaction.Means) error {
+	// TODO have a way to get the previous interaction and then update with this as the preferred mode
```

Comment:
Do we want to do this right now? I think we should make an issue to add a REST API to _update_ interaction, not just overwrite it.

Reply:
Made the issue: https://github.com/joshprzybyszewski/cribbage/issues/21